src/gap.c

```diff
@@ -876,6 +876,36 @@ Obj FuncRUNTIMES( Obj     self)
 }
 
 
+
+/****************************************************************************
+**
+*F  FuncMicrosecondsSinceEpoch( <self> )
+**
+**  'FuncMicrosecondsSinceEpoch' returns the current time since 1.1.1970 in
+**  microseconds.
+**
+**  The accuracy of this is system dependent.
+**
+**  Note that gettimeofday has been marked obsolete in the POSIX standard,
+**  we are using it because it is implemented in most systems still
+**  and will add clock_gettime in due course.
+*/
```

Comment:
Will we definitely add `clock_gettime` in due course? Isn't it more like this:
"We are using it anyway because most systems implement it, unlike its
successor clock_gettime. If the need should arise, we can add different
implementations using clock_gettime or other APIs (e.g. for Windows)."